src/pages/home/library/EditSong.tsx

```diff
@@ -1,12 +1,13 @@
-import { Box, Stack, Tab, Tabs, Theme, Typography } from "@mui/material";
+import { Stack, Tab, Tabs, Theme, Typography } from "@mui/material";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import DeleteIcon from "@mui/icons-material/Delete";
```

Comment:
Using icons from MUI vs uploading them manually.

I think that for the foreseeable future we should use icons from MUI if possible, because there are a lot of changes happening on the design end that makes it difficult to maintain our internal icons. Once we have a more stable process we can upload our own icons and maintain that. I dont think spending the extra effort is worth it at the moment.